client/src/app/site/agenda/services/agenda-import.service.ts

```diff
@@ -8,18 +8,17 @@ import { TopicRepositoryService } from 'app/core/repositories/topics/topic-repos
 import { BaseImportService, NewEntry } from 'app/core/ui-services/base-import.service';
 import { DurationService } from 'app/core/ui-services/duration.service';
 import { ItemVisibilityChoices } from 'app/shared/models/agenda/item';
-import { ViewCreateTopic } from 'app/site/topics/models/view-create-topic';
 import { CreateTopic } from '../../topics/models/create-topic';
 
 @Injectable({
     providedIn: 'root'
 })
-export class AgendaImportService extends BaseImportService<ViewCreateTopic> {
+export class AgendaImportService extends BaseImportService<CreateTopic> {
```

Comment:
Please rename that class. This service imports topics, not an agenda. Importing an agenda would be a tremendous amount of unclear tasks